content/ux-guide/research/ethics.md

```diff
@@ -0,0 +1,67 @@
+
+## Respect
+
+We respect the people who participate in our research and acknowledge  
+that they are experts on their own experiences.
+
+### Ways to demonstrate respect:
+
+- Seek permission from potential research participants as early as possible, and avoid consuming their time before they’ve formally opted-in
+- Meet at times and locations convenient for your research participants
+- Obtain participants' [informed consent]({{ '../../research/do/#getting-informed-consent' | url }}); that is, ensure that anyone who participates in your research does so of their own free will, and that they have enough information to make the decision to participate responsibly
+- If someone is uncomfortable with or prefers not to sign a participant agreement, offer them alternative opportunities to participate, or the option to not participate at all
+- Honor people’s opinions and choices. Do not correct participants during a research session. You are there to develop empathy and learn about people’s experiences as they experience them, so there are no right or wrong answers to your interview questions.
+- If someone who participates in your research appears uncomfortable, offer that person the option to skip the subject altogether or to cease their participation
+- Ask yourself: am I treating people as collaborators in, rather than subjects of, the research process?
+
+
+## Responsibility
+
+We have a responsibility to further the best interests of the people and the country we collectively serve. We avoid actions that might bring harm to our participants.
+
+### Ways to demonstrate responsibility:
+
+- Consider potential risks to participants in your research planning. Ensure that your research questions are ethical and come up with research plans that will bring no harm to participants
+- Obtain [informed consent]({{ '../../research/do/#getting-informed-consent' | url }})
+- Never record people without their consent
+- Work to ensure your potential research participants are not impacted negatively if they don’t participate in your research
+- Account for diversity and inclusion in your recruiting (for example, by specifically recruiting people who use a screen reader to navigate your website)
+- [Protect participant privacy]({{ '../../research/privacy/' | url }})
+
+
+## Honesty
+
+Design research should never be covert or manipulative. We provide clear descriptions of our data practices in our [Privacy Impact Assessment for Design Research [GSA.gov]](https://www.gsa.gov/cdnstatic/20181022%20-%20Design%20Research%20PIA_posted%20version.pdf).
+
+### Ways to demonstrate honesty:
+
+- Acknowledge that all research is subject to [bias]({{ '../../research/bias/' | url }}), and actively work to counter it
```

Comment:
Nothing you did here, but something I can't help but want to adjust as a product designer is the "it" on its own line here. Could make this
`counter&nbsp;it`